README.md

```diff
@@ -1,2 +1,256 @@
-# hZookeeper
-A Testbed using hydra for testing Zookeeper at scale.
+#hZookeeper
+
+##Background:
+
+###Mesos/Marathon: 
```

Comment:
All of this is some great documentation. I think it is general enough to be put into a separate docs repo/folder under lake-lerna ? or may be move to hHelloWorld.
@sushilks   what do you think ?